libcxx/include/cmath

```diff
@@ -765,6 +766,54 @@ _LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR_SINCE_CXX20 _Tp __constexpr_scalbn(_Tp _
   return __builtin_scalbn(__x, __exp);
 }
 
+#if _LIBCPP_STD_VER >= 17
```

Comment:
This should probably be in `include/__math/special_functions.h`.

Reply:
I raised similar thoughts in a [previous comment](https://github.com/llvm/llvm-project/pull/89982#discussion_r1581533949). 
Is it valid to use the `__math/` folder instead of a newly created `__cmath/` folder? Note, that the special math functions are only part of the `<cmath>` and not `<math.h>` header. 
In general, I am more than in favor to move that code into a separate header.

Reply:
Yes, that's fine. It's been named `__math` instead of `__math_h` so it makes sense to move any math code in there.